jsapp/js/assetUtils.ts

```diff
@@ -520,21 +521,26 @@ export function injectSupplementalRowsIntoListOfRows(
   // on Back end, to build a list of columns grouped by source question
   const additionalFields = asset.analysis_form_json?.additional_fields || [];
   const extraColsBySource: Record<string, AnalysisFormJsonField[]> = {};
-  additionalFields.forEach((add_field: AnalysisFormJsonField) => {
-    const sourceName: string = add_field.source;
+  additionalFields.forEach((field: AnalysisFormJsonField) => {
+    // We don't want to display notes in the UI, so we omit it here:
```

Comment:
Maybe a bit more verbose of a comment?

Reply:
Rewrote it here and in `submissionUtils.ts` too